### ISSUE TITLE
Reporting on use

### ISSUE THREAD

**Issue by MattHJensen** [OpenSourcePolicyCenter/webapp-public#465]
It would be helpful to have a log of ospc.org users, including IP address, IP owner, and pages visited. 

Additionally, it would be helpful to have a list of all TaxBrain and CCC runs with run number and IP address, and IP owner.

**Comment by MattHJensen** [comment]
@brittainhard, the top priority is to check that we are currently recording and _saving_ logs. The reporting could come afterwards.  It would be great to be sure that we are saving logs before we start our marketing so we can look back on what worked and didn't work.

**Comment by brittainhard** [comment]
@MattHJensen we are using a heroku app called papertrail to do our logging. It logs IPs and websites visited. It saves logs for 7 days and deletes them, but we can save all the logs by connecting an S3 bucket. From there I can write a script that filters the log output to look for IPs and page visits.

**Comment by MattHJensen** [comment]
This is resolved and @GoFroggyRun is accessing the logs and will document how to access the logs. If we need to add any more info to the logs, we can have a new issue. Closing.
